src/main/java/com/iprd/fhir/utils/Utils.java

```diff
@@ -481,4 +484,25 @@ public static byte[] zipStringData(String s) {
 			return null;
 		}
 	}
+
+	// Don't remove this function. This may be utilised later
+	public static List<String> getKeysForMapCache(List<String> allClinics, String from, String to){
+		ArrayList<String> categories = new ArrayList<String>(Arrays.asList("antenatal", "postnatal", "child-birth", "family-planning"));
```

Comment:
Add constants for these strings

Reply:
this is not being utilized. creating string constants for this will occupy unnecessary memory.

Reply:
Comment out the code if it is not used in this version

Reply:
remove code if not being used. If it will be used later then uncomment